rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1088,14 +1088,15 @@ private boolean isSvpOngoing() {
     }
 
     private void processSvpFundTransactionUnsigned(Keccak256 rskTxHash, Federation proposedFederation) {
-        Coin spendableValueFromProposedFederation = bridgeConstants.getSpendableValueFromProposedFederation();
         try {
-            BtcTransaction svpFundTransactionUnsigned = createSvpFundTransaction(proposedFederation, spendableValueFromProposedFederation);
+            BtcTransaction svpFundTransactionUnsigned = createSvpFundTransaction(proposedFederation);
             provider.setSvpFundTxHashUnsigned(svpFundTransactionUnsigned.getHash());
             PegoutsWaitingForConfirmations pegoutsWaitingForConfirmations = provider.getPegoutsWaitingForConfirmations();
 
             List<UTXO> utxosToUse = federationSupport.getActiveFederationBtcUTXOs();
-            settleReleaseRequest(utxosToUse, pegoutsWaitingForConfirmations, svpFundTransactionUnsigned, rskTxHash, spendableValueFromProposedFederation);
+            // minPegoutValue to proposed fed, minPegoutValue to flyover proposed fed
+            Coin totalValueSentToProposedFederation = bridgeConstants.getMinimumPegoutTxValue().multiply(2);
```

Comment:
Can we get this value from the transaction inputs? Feel like it will be more robust that way, a change in the transaction will automatically be reflected here